draft-ietf-idr-linklocal-capability.xml

```diff
@@ -50,51 +50,71 @@
     <area>Routing</area>
     <workgroup>Interdomain Routing</workgroup>
     <abstract>
-      <t>BGP <xref target="RFC4271" format="default"/>, was originally designed
-to provide reachability between domains and between the edges of a domain.
-As such, BGP assumes the next hop towards any reachable destination may
-not reside on the advertising speaker, but rather may either be through
-a router connected to the same subnet as the speaker, or through a router
-only reachable by traversing multiple hops through the network. Because
-of this, as per <xref target="RFC4291" format="default"/> - BGP does not recognize  IPv6 link-local addresses,
-as a valid next hop for the forwarding purposes.</t>
-      <t>This draft standardizes the operation of BGP over a point-to-point
-link using link-local IPv6 addressing only.</t>
+      <t>
+        BGP <xref target="RFC4271"/>, was originally designed
+        to provide reachability between domains and between the edges of a domain.
+      </t>
+      <t>
+        To support IPv6 reachability, BGP relies heavily on its Multiprotocol Extensions
+        as defined in <xref target="RFC2545"/>, which is crucial for
+        enabling BGP-4 to advertise IPv6 routes alongside IPv4. This extension not only
+        permits the exchange of IPv6 routing information but also establishes the structure
+        for IPv6 next hop handling within BGP updates.
+        As such, BGP assumes the next hop towards any reachable destination may
+        not reside on the advertising speaker, but rather may either be through
+        a router connected to the same subnet as the speaker, or through a router
+        only reachable by traversing multiple hops through the network.
+        <xref target="RFC2545"/> introduced the ability to advertise a global IPv6
+        address as the BGP next hop. When the advertising system is directly attached,
+        it may include both a global and a link-local address or only a global address,
+        when next hop length is set to 16 bytes.
+        This document updates the specification to clarify the encoding of the BGP next hop
```

Comment:
I'd suggest splitting the "This document updates" into a new paragraph to help catch the eye better.